cldf/languages.csv

```diff
@@ -1,42 +1,42 @@
 ID,Name,Glottocode,Glottolog_Name,ISO639P3code,Macroarea,Latitude,Longitude,Family
-1,Swahili,swah1253,,swh,,,,
-2,Iraqw,iraq1241,,irk,,,,
```

Comment:
can we try keep the ISO codes too please?

Reply:
Oh sorry, I removed them and the Tzotzil correction when playing with multiple profiles. I'll add them now.